public/views/aboutus.html

```diff
@@ -0,0 +1,61 @@
+ <div class="bkg">
+ <div class="container">
+    <nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header">
```

Comment:
@andela-oakinrele To make our code `DRY` i need you to extract the navigation to a different file  and include the file where needed. Google how to use `include` in Angular JS,